stylesheets/commons/Banner.less

```diff
@@ -144,6 +144,12 @@
 		}
 	}
 
+	.wiki-wildcard & {
+		@media ( min-width: 768px ) {
+			background: url( https://liquipedia.net/commons/images/c/c0/Wildcard-bg.webp ) no-repeat center / cover;
```

Comment:
not sure we should use webp files for this ...

Reply:
webp is great, high quality, lower bitesize